public/llms-full.txt

```diff
@@ -1124,58 +1124,84 @@ tableOfContents: false
 
 import { Tabs, TabItem } from '@astrojs/starlight/components';
 
-The Daytona SDK provides official Python and TypeScript interfaces for interacting with Daytona, enabling you to programmatically manage development environments and execute code.
+The Daytona SDK provides official Python and TypeScript interfaces for interacting with Daytona,
+enabling you to programmatically manage development environments and execute code.
 
 ### Quick Start
 
-<Tabs>
-  <TabItem label="Python" icon="seti:python">
+Run your first line of code in a Daytona Sandbox.
+
+#### 1. Get Your API Key
+
+👉 Need an account? [Sign up for access](https://www.daytona.io/waitlist)
 
+- Go to the Daytona [Dashboard](https://app.daytona.io/dashboard).
+- Create a new [API key](https://app.daytona.io/dashboard/keys). Make sure to save it securely,
+as it won't be shown again.
+- You'll need it in the next step.
+
+#### 2. Install the SDK
+
+<Tabs syncKey="language">
+  <TabItem label="Python" icon="seti:python">
   ```bash
   pip install daytona_sdk
   ```
+  </TabItem>
+  <TabItem label="TypeScript" icon="seti:typescript">
+  ```bash
+  npm install @daytonaio/sdk
+  ```
+  </TabItem>
+</Tabs>
+
+#### 3. Write Your Code
 
+<Tabs syncKey="language">
+  <TabItem label="Python" icon="seti:python">
+  Create a file named: `main.py`
   ```python
   from daytona_sdk import Daytona, DaytonaConfig
 
-  # Define the configuration
+# Define the configuration
+
   config = DaytonaConfig(
     api_key="your-api-key",
-    server_url="your-server-url",
+    server_url="https://app.daytona.io/api",
     target="us"
   )
 
-  # Initialize the Daytona client
+# Initialize the Daytona client
+
   daytona = Daytona(config)
 
-  # Create the Sandbox instance
+# Create the Sandbox instance
+
   sandbox = daytona.create()
 
-  # Run the code securely inside the Sandbox
+# Run the code securely inside the Sandbox
+
   response = sandbox.process.code_run('print("Hello World from code!")')
   if response.exit_code != 0:
     print(f"Error: {response.exit_code} {response.result}")
   else:
       print(response.result)
   
+# Clean up
+
   daytona.remove(sandbox)
-  ```
 
+  ```
   </TabItem>
-
   <TabItem label="TypeScript" icon="seti:typescript">
-
-  ```bash
-  npm install @daytonaio/sdk
-  ```
-
+  Create a file named: `index.mts`
   ```typescript
   import { Daytona } from '@daytonaio/sdk';
 
   // Initialize the Daytona client
   const daytona = new Daytona({
     apiKey: 'your-api-key',
-    serverUrl: 'your-server-url',
+    serverUrl: 'https://app.daytona.io/api',
```

Comment:
I suggest we remove all the optional params and only leave apiKey. It's cleaner in my opinion